scipy/optimize/_minpack_py.py

```diff
@@ -620,7 +620,7 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
         Initial guess for the parameters (length N). If None, then the
         initial values will all be 1 (if the number of parameters for the
         function can be determined using introspection, otherwise a
-        ValueError is raised).
+        ValueError is raised). Note that `x0` may be used interchangeably with `p0`.
```

Comment:
With the change in error message is this statement still needed?

Reply:
I would remove it. To be on the safe side I committed both a change in the ValueError message and in the documentation. Shall we wait for other opinions?

Reply:
+1 for removing, I think the error message is good now.

Reply:
```suggestion
        ValueError is raised).
```